build.gradle

```diff
@@ -48,8 +49,8 @@ project.ext.externalDependency = [
     'kafkaAvroSerde': 'io.confluent:kafka-streams-avro-serde:5.2.2',
     'kafkaClients': 'org.apache.kafka:kafka-clients:2.3.0',
     'kafkaSchemaRegistry': 'io.confluent:kafka-schema-registry-client:3.3.1',
-    'kafkaSerializers': 'io.confluent:kafka-avro-serializer:3.3.1',
-    'kafkaStreams': 'org.apache.kafka:kafka-streams:2.3.0',
+    'kafkaSerializers': 'io.confluent:kafka-avro-serializer:5.3.2',
```

Comment:
What prompted this major version update? Would it cause any problem?

Reply:
Not all versions are present in LinkedIn artifactory, we are trying to keep the internal build.gradle file in sync with the oss one.

Reply:
Got it. Please verify this end-to-end to make sure that it doesn't cause any issue. Thanks.